deepinv/tests/test_models.py

```diff
@@ -104,6 +104,25 @@ def test_denoiser(imsize, device, denoiser):
     assert x_hat.shape == x.shape
 
 
+def test_swinir_denoiser(device):
```

Comment:
Modified because of shape constraints, but the changes I've requested should make it possible to move it back to how it was previously.